fix: strip leading and trailing whitespace from QR Code data

michaelsbradleyjr opened the pull request:
fixes #10306.

see also https://github.com/status-im/status-react/pull/10280#issuecomment-611374756.

### Summary

All leading and trailing whitespace is now removed from strings derived from QR codes, both in the case of `get-qr-code-data` (real camera) and `qr-test-view` (mock camera).

#### Platforms
<!-- (Optional. Specify which platforms should be tested) -->

- Android
- iOS

#### Areas that maybe impacted
<!-- (Optional. Specify if some specific areas has to be tested, for example 1-1 chats) -->

##### Functional

- mailservers

### Steps to test
<!-- (Specify exact steps to test if there are such) -->

- Use a QR code generator that will preserve leading and trailing whitespace in the text to be encoded, and generate a QR code that has e.g. leading and/or trailing spaces and new lines. On macOS you can use `qrencode` (`brew install qrencode`).
- Open Status
- Navigate to Profile -> Sync settings -> Mailserver
- Click the `+` in the top-right of the screen.
- Click the QR code button-icon in the second field that's titled *Mailserver address*.
- Depending on the `QR_READ_TEST_MENU` flag in your development setup, either enter some text that has leading/trailing whitespace, or take a picture of the QR code you generated in the first step.
- When you are returned to the *Add mailserver* screen notice that the leading/trailing whitespace has been stripped away.

status: ready

status-im-auto commented:
### Jenkins Builds
| :grey_question: | Commit | :hash: | Finished (UTC) | Duration | Platform | Result |
|-|-|-|-|-|-|-|
| :heavy_check_mark: | 7c65ebe1 | [#1](https://ci.status.im/job/status-react/job/prs/job/android-e2e/job/PR-10332/1/) | 2020-04-14 20:56:19 | ~8 min | `android-e2e` | [:package:`apk`](https://status-im-prs.ams3.digitaloceanspaces.com/StatusIm-200414-204738-7c65eb-x86.apk) [:calling:](https://chart.apis.google.com/chart?cht=qr&chs=400x400&chld=L%7C%0A1&chl=https%3A%2F%2Fstatus-im-prs.ams3.digitaloceanspaces.com%2FStatusIm-200414-204738-7c65eb-x86.apk)|

flexsurfer commented on src/status_im/ui/components/camera.cljs:
```diff
@@ -34,4 +35,4 @@
 
 (defn get-qr-code-data [code]
-  (.-data code))
+  (string/trim (.-data code)))
```
```suggestion
  (when (.-data code) (string/trim (.-data code))))
```

michaelsbradleyjr replied:
I updated in both places to use `when-let`, which carries an assumption (I think) that `nil` is okay to pass on even if `(.-data code)` or `@text-value` was `false`. Is it acceptable?

flexsurfer reviewed (changes requested):
string functions are not null safe

michaelsbradleyjr committed:
fix: strip leading and trailing whitespace from QR Code data

All leading and trailing whitespace is now removed from strings derived from QR
codes, both in the case of `get-qr-code-data` (real camera) and
`qr-test-view` (mock camera).

Signed-off-by: Andrey Shovkoplyas <motor4ik@gmail.com>

status-im-auto commented:
## 99% of end-end tests have passed
```
Total executed tests: 91
Failed tests: 1
Passed tests: 90
```
<h3>Failed tests (1)</h3><details><summary>Click to expand</summary><br/><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/1334251">test_wallet_set_up</a></b></td></tr><tr><td><p><blockquote><div>Device 1: Looking for an element by text part: 'These three words prove that a transaction is safe.'</div><div>Device 1: Looking for an element by text part: 'You should see these words before signing each transaction. If you don't, cancel and sign out.'</div></blockquote></p><code> 'These three words prove that a transaction is safe.' text is not displayed
     'You should see these words before signing each transaction. If you don't, cancel and sign out.' text is not displayed
     'These three words prove that a transaction is sa</code><br/><br/><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/b2f22653ab9c47f68980dfb9650a9212?auth=c196c2af1fb76c26716155fba89015d9">Steps, video, logs</a></li><li><a href="https://assets.saucelabs.com/jobs/b2f22653ab9c47f68980dfb9650a9212/0037screenshot.png?auth=c196c2af1fb76c26716155fba89015d9">Failure screenshot</a></li></ul></p></ul></p></td></tr></tbody></table></details><h3>Passed tests (90)</h3><details><summary>Click to expand</summary><br/><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/1334346">test_decline_transactions_in_1_1_chat</a></b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/8338a7576a4c43ad85c8b3f811ab1fdf?auth=4546dc551785229ce621498f400a1c4c">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://saucelabs.com/jobs/049105e468af48fe994599c53b7dbee3?auth=11ecf0133a477850e89aa86422a633f6">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/1334281">test_delete_chats_via_delete_button</a></b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/0f187e1f1c17463cb58c9eab61d5c03e?auth=264f532d1600976244f6db03872cbf9a">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>3. <a href="https://ethstatus.testrail.net/index.php?/tests/view/1334237">test_can_see_all_transactions_in_history</a></b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/e4e8d62483d84f439d3cb0e93dda342c?auth=1e5278b0283c8263459daf734b7c23e8">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>4. <a href="https://ethstatus.testrail.net/index.php?/tests/view/1334319">test_install_pack_and_send_sticker</a></b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/b993bf29840a49288512677d35205e61?auth=1ead7816064beafa329824102b3c7915">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>5. <a href="https://ethstatus.testrail.net/index.php?/tests/view/1334235">test_login_with_new_account</a></b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/ae681a2f797f4c659082c4e4bfd66643?auth=27f80448721a934c70366c95e6917086">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>6. <a href="https://ethstatus.testrail.net/index.php?/tests/view/1334280">test_timestamp_in_chats</a></b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/2fd11648a8034b9cb9a09bb31d34ea81?auth=ebdf833d8e3fbd4cc921fc670e1293be">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://saucelabs.com/jobs/f5bf32fdac37416a9f7c727daa729431?auth=9ade28da90236355e0ce03b3def12f83">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>7. <a href="https://ethstatus.testrail.net/index.php?/tests/view/1334295">test_password_in_logcat_sign_in</a></b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/a020dfc9a99c457e96626bfeb0fd1561?auth=dac3967d6a17a24f98cf77a0c2976c6c">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>8. <a href="https://ethstatus.testrail.net/index.php?/tests/view/1334271">test_logcat_recovering_account</a></b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/060fd5bec60d488484debba9e42a8820?auth=ea5921549955b4f1347113368be5fef2">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>9. <a href="https://ethstatus.testrail.net/index.php?/tests/view/1334265">test_modify_transaction_fee_values</a></b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/c4701a5eae65446fa5d5ab95c1e0a311?auth=d8f256caba44929ed4b5b25e2ce6ebe7">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>10. test_public_chat_clear_history</b> (TestRail link is not found)</td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/2b83bdfef44f44f19e5dbeee8ff1c291?auth=02e512df45bfa262a81d157ac0b49826">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://saucelabs.com/jobs/e86cb496cd7341529c60820dd07cff0a?auth=a69459ea02c3e78dc6911bea75f774a8">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>11. <a href="https://ethstatus.testrail.net/index.php?/tests/view/1334284">test_need_help_section</a></b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/d295f889c4a0491eb0d09861a54c3d65?auth=3f3d704077de60f004bee90217965ae2">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>12. <a href="https://ethstatus.testrail.net/index.php?/tests/view/1334273">test_send_and_open_links</a></b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/1894606ec55247eda5182364d4dbe584?auth=e989c0f218ccfcaeface764d7aa37729">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://saucelabs.com/jobs/5add5392fc18455e944dbd918c364108?auth=fe071c346885b54b21c0b7cb5f029c4f">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>13. <a href="https://ethstatus.testrail.net/index.php?/tests/view/1334333">test_mobile_data_usage_popup_stop_syncing</a></b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/d893372ce9b348f4a4f199797a68d316?auth=40d92581c6e91de4d3cd28372b51debc">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>14. <a href="https://ethstatus.testrail.net/index.php?/tests/view/1334334">test_mobile_data_usage_settings</a></b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/e04d6677d7244ebbb7d7642d37950480?auth=31511628de1ab06d4aaccb8719d2bd8e">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>15. <a href="https://ethstatus.testrail.net/index.php?/tests/view/1334225">test_create_new_group_chat</a></b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/dd6bd237224f4021b73c6037014c229d?auth=157162186e048fcd6cc19cc2597e23c7">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://saucelabs.com/jobs/6ea7a40259f94649bce33743caf8ce74?auth=67ff0b822e0ccb0b2cbaecb37a2fefc2">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>16. <a href="https://ethstatus.testrail.net/index.php?/tests/view/1334321">test_sign_typed_message</a></b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/f3333b708c8644168d8652c69b5522d8?auth=9e7a9939cf1bb5e87e64916def929e65">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>17. <a href="https://ethstatus.testrail.net/index.php?/tests/view/1334228">test_user_can_switch_network</a></b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/0e742fa94ef34117992fce1d5329e1f5?auth=e20761b7142512bf74fcb5dffe94e96b">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>18. <a href="https://ethstatus.testrail.net/index.php?/tests/view/1334311">test_mobile_data_usage_popup_continue_syncing</a></b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/edfc977842b64854a486e0f7cac7e228?auth=6dae374a64efb27e5757b96238f6ece8">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>19. <a href="https://ethstatus.testrail.net/index.php?/tests/view/1334276">test_home_view</a></b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/ff2798abdd8349489abb1909db0a5804?auth=f6800a85e951974b8a89720259f96840">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>20. <a href="https://ethstatus.testrail.net/index.php?/tests/view/1334242">test_open_google_com_via_open_dapp</a></b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/9059ac7eeb9a434cb74e9568732c3caf?auth=2fd301b3c9cb049a8c7e93b125e775b4">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>21. <a href="https://ethstatus.testrail.net/index.php?/tests/view/1334238">test_send_message_to_newly_added_contact</a></b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/5716628e715549508cd1ca06fbc8204f?auth=ba04645e39445a84693e7f3a55387cb4">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://saucelabs.com/jobs/7189c9d73d384d7caaac9ecb7f199c51?auth=2c633b138d276f49bc5bc91e6ff97326">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>22. <a href="https://ethstatus.testrail.net/index.php?/tests/view/1334244">test_share_contact_code_and_wallet_address</a></b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/cd320b4e3d104c1e8634207e656d2f0e?auth=7a90708079ffebe4a618066fa8f6df03">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>23. <a href="https://ethstatus.testrail.net/index.php?/tests/view/1334289">test_connection_is_secure</a></b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/0a88b4b3e1d94a8197b593575068c642?auth=f7cf218e051921be55155260b56aa6e8">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>24. <a href="https://ethstatus.testrail.net/index.php?/tests/view/1334239">test_add_to_contacts</a></b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/bfdaba6653cf4d55886422b1b180170c?auth=69206cadece1a48d9fd74f94c288f1fd">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://saucelabs.com/jobs/03025a6b18cd433fb73ae75392851805?auth=62d233d324adbbd3146ad5531883b26d">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>25. <a href="https://ethstatus.testrail.net/index.php?/tests/view/1334348">test_add_account_to_multiaccount_instance_private_key</a></b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/f4d726ff76cb4dca98a55f1f1c0cae73?auth=8255e10f9af6b4da21add66bf037db5a">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>26. <a href="https://ethstatus.testrail.net/index.php?/tests/view/1334261">test_refresh_button_browsing_app_webview</a></b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/36935c4455924229ac4122df7f850929?auth=f26076ab30db5b1e52f65d4ce97ba749">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>27. <a href="https://ethstatus.testrail.net/index.php?/tests/view/1334301">test_can_add_existing_ens</a></b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/df97e9e41f234ef3ba513932e5101a18?auth=a9e7fe21c6404075eb46881987b3681e">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>28. <a href="https://ethstatus.testrail.net/index.php?/tests/view/1334292">test_transaction_wrong_password_wallet</a></b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/7b801d06755e488c88e973c201e764db?auth=0d312416138fe847769ccc22428628dc">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>29. <a href="https://ethstatus.testrail.net/index.php?/tests/view/1334266">test_unread_messages_counter_public_chat</a></b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/615997908f524f92a30dcfb2b456610a?auth=dcc6be388ffd3eba674aa147c58f899e">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://saucelabs.com/jobs/ec0fd1f4f4cb479c96cf850f7563d0ea?auth=0e3334b80ffa8d6df7901284a03d8e74">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>30. <a href="https://ethstatus.testrail.net/index.php?/tests/view/1334231">test_text_message_1_1_chat</a></b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/4eda5640c10e4a1a924c9705af96e829?auth=d04cfd4a73a809c7ad5102f05e31ca56">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://saucelabs.com/jobs/06a77bc19f574db0bc67d28000e9eff7?auth=5e44d7aa9a86066cfa057c5caed75bc2">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>31. <a href="https://ethstatus.testrail.net/index.php?/tests/view/1334320">test_can_use_purchased_stickers_on_recovered_account</a></b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/cedf86d68d934bc1ab90d1dc51304e73?auth=af3152becd99c99fa2994da9b06e2811">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>32. <a href="https://ethstatus.testrail.net/index.php?/tests/view/1334279">test_open_transaction_on_etherscan</a></b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/1c0248f1bcf94c3cb277896965187c7c?auth=26644fb822c5563b2b811e1bd2982318">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>33. <a href="https://ethstatus.testrail.net/index.php?/tests/view/1334304">test_onboarding_screen_when_requesting_tokens_for_recovered_account</a></b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/66431efe49484d728287c2e7bd980f23?auth=ecb677b54d3bce898a837eba9c80e9f4">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>34. <a href="https://ethstatus.testrail.net/index.php?/tests/view/1334278">test_contact_profile_view</a></b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/bacade30b16d46b79b66c91e8ab4a163?auth=cf6cbe30c3db9b37469907e150596afe">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>35. <a href="https://ethstatus.testrail.net/index.php?/tests/view/1334226">test_offline_add_new_group_chat_member</a></b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/bbb8ec97dce24364aeb8773a6a85e8cf?auth=ecebcd95cd4e58447625eb6c5df507cc">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://saucelabs.com/jobs/373c07c9199d4a2da75e6ea5032dc2f1?auth=95b5137d34d0cf6a7759de6b3c467e69">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>36. <a href="https://ethstatus.testrail.net/index.php?/tests/view/1334298">test_logcat_backup_recovery_phrase</a></b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/4d777481d99043e0a3cbd9ec8b825a79?auth=da8302dfee4791f2234fc8f9c55cd5b9">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>37. <a href="https://ethstatus.testrail.net/index.php?/tests/view/1334290">test_start_chat_with_ens</a></b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/1ec5a6b67cfa4fd78fa2310bc770d9e0?auth=e592c7b0cf8d4c1765976605400f2114">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>38. <a href="https://ethstatus.testrail.net/index.php?/tests/view/1334249">test_add_and_remove_contact_from_public_chat</a></b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/91fb46426c764427b36e7fa64c30b072?auth=e9f7de594d1851976b4b2a94d0f8edb9">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://saucelabs.com/jobs/520da0ff735b4816b93026d65393b1bc?auth=98e82f0edca6f212d10c916088b49f84">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>39. <a href="https://ethstatus.testrail.net/index.php?/tests/view/1334347">test_add_account_to_multiaccount_instance_seed_phrase</a></b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/2ed43eda35e54e618dc441793b0f5315?auth=2c57aea5b11ed59afc533b267fa7a21d">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>40. <a href="https://ethstatus.testrail.net/index.php?/tests/view/1334326">test_open_blocked_site</a></b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/3471bf6b04964c64bf1b28ff5f45b7a9?auth=11da2e4add3b7a390f000fb213390689">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>41. <a href="https://ethstatus.testrail.net/index.php?/tests/view/1334230">test_open_chat_by_pasting_public_key</a></b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/b81bfe27c37146279bc56f1ede0ec879?auth=0a12cf9a57d7c726480ee9abc3defd4d">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>42. <a href="https://ethstatus.testrail.net/index.php?/tests/view/1334317">test_pair_devices_sync_one_to_one_contacts_public_chat</a></b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/be6e105ca07a41ed9390fac800896049?auth=62bf0eca11aa09a280d45e9beca0f73e">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://saucelabs.com/jobs/fb5342abd35d4e64a6d346e60906c461?auth=a0d9fbd6564ec2e1c6568ae57296d90b">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>43. <a href="https://ethstatus.testrail.net/index.php?/tests/view/1334291">test_ens_username_recipient</a></b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/18f73bd5d3bf4373b0381c2e065eaede?auth=2d17c3a6ddac4dd2c446497666ea3538">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>44. <a href="https://ethstatus.testrail.net/index.php?/tests/view/1334338">test_token_with_more_than_allowed_decimals</a></b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/0802f070184743428772a11af6527037?auth=eb46a19b4fe105942a8aa31d8a88ebd7">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>45. <a href="https://ethstatus.testrail.net/index.php?/tests/view/1334323">test_fetch_more_history_in_empty_chat</a></b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/145537aec11645408dcdba2fc6fd6821?auth=a5e0d16a6f5edc43c5a30452923259a5">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>46. <a href="https://ethstatus.testrail.net/index.php?/tests/view/1334287">test_open_public_chat_using_deep_link</a></b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/44572da9d9194a71ade5c697c6a3e4ba?auth=0dab71e1978caf437600669aca5bafa1">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>47. <a href="https://ethstatus.testrail.net/index.php?/tests/view/1334262">test_switch_users_and_add_new_account</a></b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/06812d1599ee44a2a11576442c50cf6b?auth=eaf6f3b14a52b01c1bcbcd9441b67407">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>48. <a href="https://ethstatus.testrail.net/index.php?/tests/view/1334330">test_add_account_to_multiaccount_instance_generate_new</a></b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/053235724e26417e879e9aaf400f39e6?auth=05ce0e52620b1cfb8f831ce10d9979f8">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>49. <a href="https://ethstatus.testrail.net/index.php?/tests/view/1334275">test_copy_contact_code_and_wallet_address</a></b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/747f1e04770a42d08844b04f4d66aaa9?auth=61758d7e838695b486a6a95070b2d001">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>50. <a href="https://ethstatus.testrail.net/index.php?/tests/view/1334233">test_send_transaction_from_daap</a></b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/ea72be46ab1e4cf5aa331f62b9e83a17?auth=42bdf10a87159d6bc0bbdcf2fa89f061">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>51. <a href="https://ethstatus.testrail.net/index.php?/tests/view/1334350">test_send_eth_to_ens</a></b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/f166049c1f944facb2d18cd38f96dd70?auth=790090916f3d17f7bb1d742f717bf98d">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>52. <a href="https://ethstatus.testrail.net/index.php?/tests/view/1334313">test_send_two_transactions_one_after_another_in_dapp</a></b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/21a67f727964447c8ecc4d5248102fb0?auth=b3ee5c70688d47583befb14f42e14c37">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>53. <a href="https://ethstatus.testrail.net/index.php?/tests/view/1334274">test_message_marked_as_sent_in_1_1_chat</a></b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/81c9bdbb22b54908a67126bfc5a92ef1?auth=829ba3ac3d54a088b6a1aaaa236ab394">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>54. <a href="https://ethstatus.testrail.net/index.php?/tests/view/1334283">test_long_press_delete_clear_all_dapps</a></b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/e5f795c1f5af4c7285637183a753aa37?auth=07dc45b712318e7959bf147400a25a7b">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>55. <a href="https://ethstatus.testrail.net/index.php?/tests/view/1334240">test_copy_and_paste_messages</a></b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/3b0196741b7e474dbd7100fc28bd225c?auth=d64e6a36d750a3762fb370a5b488c336">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>56. <a href="https://ethstatus.testrail.net/index.php?/tests/view/1334250">test_deploy_contract_from_daap</a></b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/ef708b033a5d44f0b28e9d21d707a503?auth=03d2a8bcc1e3fa4135ad252afc1fffb7">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>57. <a href="https://ethstatus.testrail.net/index.php?/tests/view/1334243">test_back_forward_buttons_browsing_website</a></b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/1de238ff55e2475ebd9b69467aea6b06?auth=e49ac14948d2bf039990f90292456a99">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>58. <a href="https://ethstatus.testrail.net/index.php?/tests/view/1334294">test_password_in_logcat_creating_account</a></b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/c297375efcaf4a8db1309e2d5db32594?auth=e5a6137b3f6270ced4cd515759401afc">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>59. <a href="https://ethstatus.testrail.net/index.php?/tests/view/1334324">test_add_custom_token</a></b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/f2f2eb9bb3b84e119fa7f29b2082e25b?auth=f8f42f559484c5b1ac5c7987bad5cd08">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>60. <a href="https://ethstatus.testrail.net/index.php?/tests/view/1334257">test_sign_message_from_daap</a></b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/ecf7ded6b08e443f81d1c0556391085d?auth=a6af192639595cf51dfe4f511a61ee10">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>61. <a href="https://ethstatus.testrail.net/index.php?/tests/view/1334310">test_dapps_permissions</a></b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/67a5479863fd4ab5a757f465c565e09a?auth=6b78f17ef4e4b1ca920a559109de4db7">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>62. <a href="https://ethstatus.testrail.net/index.php?/tests/view/1334288">test_request_public_key_status_test_daap</a></b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/9c97f22971034ab280e1c4794aec1477?auth=82d552ce0df64acda68205403bd0e34e">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>63. <a href="https://ethstatus.testrail.net/index.php?/tests/view/1334296">test_logcat_send_transaction_from_wallet</a></b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/b3a4770d77d04750ae29728b51dfab01?auth=cb12d8e4602fe1e6037b26b9a3b10edf">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>64. <a href="https://ethstatus.testrail.net/index.php?/tests/view/1334241">test_long_press_to_delete_chat</a></b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/12f0f45274544b38bfe2fd791f3ec329?auth=21a846dc974e7a9e2cb18fe3d608aada">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>65. <a href="https://ethstatus.testrail.net/index.php?/tests/view/1334272">test_log_level_and_fleet</a></b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/2d0713bca0e54dfeacbb387ce83bbc4d?auth=41230f33b538bfbbe0604e89f01fcb04">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>66. <a href="https://ethstatus.testrail.net/index.php?/tests/view/1334337">test_add_and_delete_watch_only_account_to_multiaccount_instance</a></b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/2cacaf22cf8446eb951a06264ae4a04f?auth=14b1f78d7598ed843bd880084d304d11">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>67. test_recover_account_from_new_user_seedphrase</b> (TestRail link is not found)</td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/b8fa86ef4b7046bea7cd0c0e2e527b02?auth=b45bc94ba7dfcbe40d365c91e257c2e8">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>68. <a href="https://ethstatus.testrail.net/index.php?/tests/view/1334343">test_send_eth_in_1_1_chat</a></b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/e2ce446f976047a68f104c3f0ffde545?auth=1372da8a57d3d248b060d62ad1706a48">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://saucelabs.com/jobs/1447c1297454401c833c4c32b9697df9?auth=b68d46593685907003956e0ac8cafa22">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>69. <a href="https://ethstatus.testrail.net/index.php?/tests/view/1334331">test_ens_in_public_and_1_1_chats</a></b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/966bc6ee70aa47ab92f364cc93e2772b?auth=38838d67b2a6b7284e8bfeada5c62d7b">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://saucelabs.com/jobs/eedb2a68a24c40ab935781257272eb84?auth=c3b608161513509fbcf8d0db255f221e">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>70. <a href="https://ethstatus.testrail.net/index.php?/tests/view/1334322">test_block_user_from_public_chat</a></b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/d30d9bb3ea2f47d3ab5ad80c6caecf4e?auth=071f0f3c2ca0bfc1e74e11af26c5e5ed">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://saucelabs.com/jobs/7056ceafaff74c928e461a2513084e47?auth=9da69f4b6d048bd9975405d0d9f26ddd">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>71. <a href="https://ethstatus.testrail.net/index.php?/tests/view/1334335">test_fetching_balance_after_offline</a></b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/d7886970ae724dd3bcc2802e1b47d534?auth=370d242039fc0adb580351cad32a3fb2">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>72. <a href="https://ethstatus.testrail.net/index.php?/tests/view/1334299">test_logcat_sign_message_from_daap</a></b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/18fb3f2106ae4799ad56fc0ec11af4a7?auth=8b39feba3dbeee8327703adccecfb86a">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>73. <a href="https://ethstatus.testrail.net/index.php?/tests/view/1334246">test_offline_status</a></b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/fc503b5ff2c54a6c9b3a2b51eb225f41?auth=1dc224babe13e228b9a628e847b71cd4">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>74. <a href="https://ethstatus.testrail.net/index.php?/tests/view/1334256">test_manage_assets</a></b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/030228b7eed34438a2ef51dedc047b45?auth=28275381fe403cee6ea9dd3ac76566a6">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>75. <a href="https://ethstatus.testrail.net/index.php?/tests/view/1334234">test_offline_messaging_1_1_chat</a></b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/9dc91c42f4ff4a3ba3d86d048e6ec589?auth=e26fc8710c914099c0bf515d22a96714">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://saucelabs.com/jobs/09f57fb7b4e3497fb1e9dcc688c03fb1?auth=b1f2a8ad7798fb065b99099a8edc5f26">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>76. <a href="https://ethstatus.testrail.net/index.php?/tests/view/1334285">test_account_recovery_with_uppercase_recovery_phrase</a></b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/ed08a8c77d684f6fa2b80b5a28a582f8?auth=153dded36d37f9dc82cdb0ba5fc696eb">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>77. <a href="https://ethstatus.testrail.net/index.php?/tests/view/1334307">test_add_new_keycard_account</a></b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/73348bd795ad46f79d2a7cad6a6d0445?auth=9619933c30b78b85644f647190f401f4">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>78. <a href="https://ethstatus.testrail.net/index.php?/tests/view/1334259">test_send_token_with_7_decimals</a></b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/feb2d8ea346541f2a24b72d208e1fe5b?auth=5eb006fc885e29d9b33e53f1d86b068f">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>79. <a href="https://ethstatus.testrail.net/index.php?/tests/view/1334277">test_user_can_see_all_own_assets_after_account_recovering</a></b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/baff5d84096c4f3a996b50c0e4fa9ddc?auth=7a96a7060a2b57700c5052cc757532ec">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>80. <a href="https://ethstatus.testrail.net/index.php?/tests/view/1334268">test_pass_phrase_validation</a></b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/83d18864c1964c959cc3318bd7f0b004?auth=0a8ab0854a063b91c6e1e1421c8511c6">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>81. <a href="https://ethstatus.testrail.net/index.php?/tests/view/1334247">test_send_emoji</a></b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/5999518d7126417abecf46a6223e9588?auth=02a6476f141925b321834980cc273885">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>82. <a href="https://ethstatus.testrail.net/index.php?/tests/view/1334260">test_filters_from_daap</a></b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/db28f52f23334d9ca293930d2622dd88?auth=375e861d9f75aa7643b1a1f4e3c16782">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>83. <a href="https://ethstatus.testrail.net/index.php?/tests/view/1334303">test_redirect_to_public_chat_tapping_tag_message</a></b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/70220b844bf64bf6b05aee7c3734b59f?auth=e8d33d175f5b8b832a5b00f6278e8b50">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>84. <a href="https://ethstatus.testrail.net/index.php?/tests/view/1334297">test_logcat_send_transaction_from_daap</a></b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/33ca91a38e3c4b32bffd69c096c75947?auth=a0962e872d7eee5eb83aeab3a1fa7ee9">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>85. <a href="https://ethstatus.testrail.net/index.php?/tests/view/1334232">test_send_eth_from_wallet_to_address</a></b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/400d9a36ac83469d8546a479a5bc77c5?auth=1f60ff84473fa91b9684705a47c388b6">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>86. <a href="https://ethstatus.testrail.net/index.php?/tests/view/1334245">test_send_stt_from_wallet</a></b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/fc30e914508e47758f32288a0037d9a6?auth=6aaf4945d8551f35ea5e6c79bed076a3">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>87. <a href="https://ethstatus.testrail.net/index.php?/tests/view/1334345">test_request_and_receive_stt_in_1_1_chat_offline</a></b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/dbb2babd0d6a41a3aff08150b4e99029?auth=a796165ea79a22fdebe3e97dd8f6e444">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://saucelabs.com/jobs/adecd6c97a9f495e820c74c5a8094510?auth=1bf7cc61c296b2a76ef971e3bd373ec0">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>88. <a href="https://ethstatus.testrail.net/index.php?/tests/view/1334253">test_messaging_in_different_networks</a></b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/693db2292d584aaba2d678ee65f6e889?auth=ca2312c671bbc91c23a51a85a414b776">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://saucelabs.com/jobs/8482ad09d089420884b033e0d875f187?auth=bfaf0248ef0a1a7a228ce9c87ea90c03">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>89. <a href="https://ethstatus.testrail.net/index.php?/tests/view/1334258">test_collectible_from_wallet_opens_in_browser_view</a></b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/cadb03e7b5db4c8fb9622a05cedeb5d6?auth=eea48e3e7328ed846404b4f690f5d888">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>90. <a href="https://ethstatus.testrail.net/index.php?/tests/view/1334293">test_insufficient_funds_wallet_positive_balance</a></b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/131cef81b7754fb39ab87949bd5994c6?auth=039bffef453547d3e09aaf1a50a084f0">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table></details>

churik commented:
Checked from:
- scanning QR from Start new chat > QR for chat key
- adding custom mailserver / bootnode
- EIP-681 links from the wallet.

In all places, whitespaces are trimmed.
Thanks!
